js/viz.js

```diff
@@ -47,16 +47,25 @@ const viz = {
 
     const linkForce = d3.forceLink(this.links);
     linkForce.id((d) => d.hostname);
-    linkForce.distance(100);
+    linkForce.distance(50);
     simulation.force('link', linkForce);
 
     const centerForce = d3.forceCenter(this.width/2, this.height/2);
     centerForce.x(this.width/2);
     centerForce.y(this.height/2);
     simulation.force('center', centerForce);
 
+    const forceX = d3.forceX(this.width/2);
+    forceX.strength(0.3);
+    simulation.force('x', forceX);
+
+    const forceY = d3.forceY(this.height/2);
+    forceY.strength(0.3);
+    simulation.force('y', forceY);
+
```

Comment:
As seen from the above gifs, before this PR, the newly added nodes started from <0,0>. It is now improved to start from position `<x,y>`. This way nodes are always clustered toward the center.